spec/collectionspace/mapper/data_mapper_core_spec.rb

```diff
@@ -141,8 +141,8 @@
     end
     context 'movement record' do
       before(:all) do
-        @loanoutmapper = get_json_record_mapper(path: 'spec/fixtures/files/mappers/release_6_1/core/core_6_1_0-movement.json')
-        @handler = DataHandler.new(record_mapper: @loanoutmapper, cache: @cache, client: core_client, config: @config)
+        @movementmapper = get_json_record_mapper(path: 'spec/fixtures/files/mappers/release_6_1/core/core_6_1_0-movement.json')
```

Comment:
@kevin-kariuki  - Changed variable name to match record type being tested. 

This branch/PR is called core-media, but there is no `context 'media record' do` section in the PR? Is that intentional?